src/widgets/index.js

```diff
@@ -824,7 +824,7 @@ UI.widgets.fieldParams[UI.ns.ui('TextField').uri] = { }
 
 UI.widgets.fieldParams[UI.ns.ui('PhoneField').uri] = { 'size': 20, 'uriPrefix': 'tel:' }
 UI.widgets.fieldParams[UI.ns.ui('PhoneField').uri].pattern =
-  /^\s*\+?[ 0-9-]+[0-9]\s*$/
+  /^\s*\+?[0-9-]+[0-9]\s*$/
```

Comment:
Shouldn't we also disallow whitespace on either side of the phone number?

Reply:
While we're at it, shall we just `\d`?

Reply:
Can do - wanted to keep the changes minimal, as to not introduce any errors (I'm not very good with regex, after all). But I've tried a slight change now, hopefully to the better.